solutions/usaco-737.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: usaco-737
+source: USACO Bronze 2017 US Open
+title: Modern Art
+author: Juheon Rhee
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_art_bronze_open17.html)
+
+## Implementation
+**Time Complexity**: $\mathcal O(N^3)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+res = [[0] * 11 for _ in range(11)]
+left, right, down, up = [0] * 10, [0] * 10, [0] * 10, [0] * 10
+
+ans = [0] * 10
```

Comment:
better names than `res` & `ans`

Reply:
wait i think these names are fine 👀

Reply:
ill change it if u give me suggestion

Reply:
what do they store? what does it mean in the context of the problem?

Reply:
ans stores basically 1 if the possible number can be the "starting number" -> the question is asking how many possible starting numbers 

and res stores the "art" or the input